src/key.ts

```diff
@@ -1,2 +1,2 @@
 export const api_key = '250b65ac19c1d23b70718726a42ee160';
-// export const PROXY_URL = 'https://cors-anywhere.herokuapp.com/';
+export const PROXY_URL = 'https://cors-anywhere.herokuapp.com/';
```

Comment:
Did calling the API work before you introduced the `PROXY_URL`? If so, what is it's purpose?

Reply:
It did, but when i switched from chrome to Firefox to test I ran into a CORS error. Its there as a "just in case".